### DISCUSSION TITLE
Gender value for the EMPI Enrichment

### DISCUSSION THREAD

**Discussion by Haneefyaseen** [qe-collaborative-services/1115-hub#70] [Q&A] [answered]
This is Mohamed from RHIO. I got a question on the gender values. Should we take SEX_AT_BIRTH_CODE or the ADMINISTRATIVE_SEX_CODE for our EMPI Enrichment process? Could someone clarify? 

Currently we are using ADMINISTRATIVE_SEX_CODE as the gender value.

Thanks.

**Comment by shah** [comment]
@reeager please investigate and respond.

**Comment by reeager** [comment, marked as answer]
Just administrative sex would be in eMPI

**Reply by Haneefyaseen** [reply]
Thanks @shah and @reeager
